Models/WishListListPart.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Orchard.ContentManagement;
+using Orchard.ContentManagement.Utilities;
+using Orchard.Environment.Extensions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Nwazet.Commerce.Models {
+    [OrchardFeature("Nwazet.WishLists")]
+    public class WishListListPart : ContentPart<WishListListPartRecord> {
+        private static readonly char[] separator = new[] { '{', '}', ',' };
+        private readonly LazyField<IEnumerable<ContentItem>> _wishListElements = new LazyField<IEnumerable<ContentItem>>();
+        public LazyField<IEnumerable<ContentItem>> WishListElementsField { get { return _wishListElements; } }
+
+        private string _serializedItemIds {
+            get { return Retrieve(r => r.SerializedIds) ?? ""; }
+            set { Store(r => r.SerializedIds, value); }
+        }
+
+        public int[] Ids {
+            get {
+                return _serializedItemIds
+                    .Split(separator, StringSplitOptions.RemoveEmptyEntries)
+                    .Select(frag => int.Parse(frag))
+                    .ToArray();
+            }
+            set {
+                _serializedItemIds = "{" + string.Join("},{", value) + "}";
```

Comment:
I got rid of the string entirely. Since I am storing the Id of the WishList in the record for the item, I figure I don't really need the string anyway.